kyaml/yaml/alias.go

```diff
@@ -19,6 +19,16 @@ const DefaultSequenceStyle = CompactSequenceStyle
 var sequenceIndentationStyle = DefaultSequenceStyle
 var indent = DefaultIndent
 
+// SetSequenceIndentationStyle sets the sequenceIndentationStyle variable
+func SetSequenceIndentationStyle(style string) {
```

Comment:
I have to think more about setting this at the Go package level, why do we need to that ?

Reply:
Yes. Added new methods MarshalWithIndent and NewEncoderWithIndent. Removed package level configurations.